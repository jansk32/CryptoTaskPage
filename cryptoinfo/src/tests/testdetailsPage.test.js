import React from 'react';
import DetailsPage from '../detailsPage'
import renderer from 'react-test-renderer';

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'), 
    useParams: () => ({
      coinId: "bitcoin"
    }),
    useRouteMatch: () => ({ url: '/details/bitcoin' }),
  }));

test("Testing a Details page", () => {
    const component = renderer.create(
        <DetailsPage />
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})