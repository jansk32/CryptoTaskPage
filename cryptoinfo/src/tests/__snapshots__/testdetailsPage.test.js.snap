// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing a Details page 1`] = `
<div>
  <center
    style={
      Object {
        "paddingTop": "10%",
      }
    }
  >
    <img
      alt="Loading..."
      className="loadingLogo"
      src="loading_logo.png"
      style={
        Object {
          "height": "20%",
          "width": "20%",
        }
      }
    />
    <h1>
      Loading...
    </h1>
  </center>
</div>
`;
